web/crux/src/app/team/user.http.controller.ts

```diff
@@ -22,6 +23,7 @@ export default class UserHttpController {
 
   @Post()
   @HttpCode(200)
+  @AuditLogLevel('disabled')
```

Comment:
Just asking, I know that was not the task in this PR, but it would be great if we create a const for the AuditLogLevels, no? Using `DISABLED` or `AUDIT_DISABLED` instead of `('disabled')`